resources/gemini-3c/main.tf

```diff
@@ -4,52 +4,53 @@ module "gemini-3c" {
   network-name    = "gemini-3c"
   bootstrap-node-config = {
     droplet_size        = var.droplet-size
-    deployment-version  = 3
+    deployment-version  = 4
     regions             = []
     nodes-per-region    = 0
     additional-node-ips = var.hetzner_bootstrap_node_ips
     docker-org          = "subspace"
-    docker-tag          = "gemini-3c-2023-jan-18"
+    docker-tag          = "gemini-3c-2023-jan-27"
     reserved-only       = false
-    prune               = false
+    prune               = true
   }
   full-node-config = {
     droplet_size        = var.droplet-size
-    deployment-version  = 3
+    deployment-version  = 4
     regions             = []
     nodes-per-region    = 0
     additional-node-ips = var.hetzner_full_node_ips
     docker-org          = "subspace"
-    docker-tag          = "gemini-3c-2023-jan-18"
+    docker-tag          = "gemini-3c-2023-jan-27"
     reserved-only       = false
-    prune               = false
+    prune               = true
   }
   rpc-node-config = {
     droplet_size        = var.droplet-size
-    deployment-version  = 4
+    deployment-version  = 5
     regions             = []
     nodes-per-region    = 0
     additional-node-ips = var.hetzner_rpc_node_ips
     docker-org          = "subspace"
-    docker-tag          = "gemini-3c-2023-jan-18"
+    docker-tag          = "gemini-3c-2023-jan-27"
     domain-prefix       = "eu"
     reserved-only       = false
-    prune               = false
+    prune               = true
   }
   farmer-node-config = {
     droplet_size           = var.droplet-size
-    deployment-version     = 1
+    deployment-version     = 5
     regions                = ["ams3"]
     nodes-per-region       = 1
     additional-node-ips    = []
     docker-org             = "subspace"
-    docker-tag             = "gemini-3c-2023-jan-18"
+    docker-tag             = "gemini-3c-2023-jan-27"
     reserved-only          = false
     plot-size              = "10G"
     reward-address         = var.farmer-reward-address
     force-block-production = false
     prune                  = false
   }
+  piece_cache_size     = "500KiB"
```

Comment:
Why 500KiB? We want something like 10G at least instead. I was talking about 500k pieces, not bytes.